common/static/common/js/discussion/utils.js

```diff
@@ -539,6 +542,37 @@
             };
         };
 
+        DiscussionUtil.handleKeypressInToolbar = function(event) {
+                var $currentButton, $nextButton, $toolbar, $allButtons,
+                    keyPressed, nextIndex, currentButtonIndex;
+
+                var leftKey = this.LEFT_KEYCODE;
+                var rightKey = this.RIGHT_KEYCODE;
+
+                $currentButton = $(event.target);
+                keyPressed = event.which || event.keyCode;
+                $toolbar = $(event.target).parent();
```

Comment:
Isn't this just `$currentButton.parent()`?

Reply:
Oh. Yes.